### ISSUE TITLE
card_pred = (pred_logits.argmax(-1) != pred_logits.shape[-1] - 1).sum(1)

### ISSUE THREAD

**Issue by maohule** [dk-liang/CLTR#37]
您好，感谢您分享的论文代码。这里有个问题想要询问一下。dataset.py中target['labels'] = torch.ones([1, num_points]).squeeze(0).type(torch.LongTensor)，这里明确把人类别标签设为1。但是在conditional_detr.py文件中，loss_cardinality函数中的代码card_pred = (pred_logits.argmax(-1) != pred_logits.shape[-1] - 1).sum(1)（第169行）。通过计算，pred_logits.shape[-1] - 1等于1。那么不应该是pred_logits.argmax(-1) == pred_logits.shape[-1] - 1).sum(1)为预测的人数吗？
